backstage-templates-for-serverless/serverless-microservice/template-serverless-microservice/terraform/main.tf

```diff
@@ -0,0 +1,233 @@
+variable "name" {
```

Comment:
As discussed, remove the source code for terraform and js source and keep it in the repository where it was originally hosted. This servers two purposes 1. you dont need to maintain it here for changes in source repo 2. the idea for integration is to keep the source code as is and just use backstage templating to use it here.

Reply:
@elamaran11 thanks for the feedback, please find below my comments before I proceed.

1. I developed this code for this template so there is no other repo to be maintained as you suggested. But no problem, if you want to keep it segregated, I can move somewhere else. 
2. The `catalog-info.yaml` in the root is already targeting the templates, so there is no need for an additional file. I have extensively validated this on backstage and it's working properly. Feel to try on your own too.
3. True, and by the way, out of 5 templates for EKS the one you linked is the only with a README file. It seems to me that the deployment, troubleshooting and clean up steps will pretty much be the same for all templates. I'm wondering if it would make more sense having these common steps in the repository root README file, and only keep specific instructions in the template READMEs.

Reply:
> @elamaran11 thanks for the feedback, please find below my comments before I proceed.
> 
> 1. I developed this code for this template so there is no other repo to be maintained as you suggested. But no problem, if you want to keep it segregated, I can move somewhere else.
> 2. The `catalog-info.yaml` in the root is already targeting the templates, so there is no need for an additional file. I have extensively validated this on backstage and it's working properly. Feel to try on your own too.
> 3. True, and by the way, out of 5 templates for EKS the one you linked is the only with a README file. It seems to me that the deployment, troubleshooting and clean up steps will pretty much be the same for all templates. I'm wondering if it would make more sense having these common steps in the repository root README file, and only keep specific instructions in the template READMEs.

@tgpadua Thankyou for responding to the feedback. Please see my comments below:
1. Yes i would appreciate if you can move it to elsewhere. You can even move it to personal repo for timebeing until you move it to AWS repos.
2. I think i got your point and its my bad. I dont think we need a new folder by name `backstage-templates-for-serverless`.  We want to stick to one folder by name `backstage-templates-for-eks` in the root of the repo. Move your `serverlesss-microservice` pattern under `backstage-templates-for-eks` folder and also update the `catalog-info.yaml` under `backstage-templates-for-eks` with your new pattern. If you see the setup instructions [here](https://github.com/cnoe-io/backstage-terraform-integrations#:~:text=add%20the%20following%20lines%20for%20catalog%20location%20under), idea is to render only one catalog as part of this integration. So bottom line is we dont need new catalog and new folde by name `backstage-templates-for-serverless`. 
3. README file is a mandatory criteria for each pattern before merge. We want to show how to select that backstage template, what fields to populate and output with screenshots. You can skip over the deployment steps like the GPU pattern README. We will have contributors working on updating the README file including me.

Please reachout for any questions. Appreciate if you can make these changes, looking forward for merging this PR.

Reply:
@elamaran11  Thanks for the additional instructions, here are some additional thoughts...

1. No problem at all, I will move those assets into a different repo. 
2. Alright, I will merge the `serverless-microservice` pattern in the existing catalog (`backstage-templates-for-eks`). I used a new catalog as this pattern doesn't deployed resources on EKS. With that said, what are you thoughts on rename it to something more generic like `backastage-templates-for-aws` or just `backstage-templates`?
3. Agreed. I will add the README for the pattern.

Reply:
Thankyou @tgpadua . Looking forward to merge the PR.

Reply:
Hey @elamaran11 it's done. Now, all assets are in a different repo, the pattern was merged into the existing catalog and a [README](https://github.com/tgpadua/backstage-terraform-integrations/tree/main/backstage-templates-for-eks/serverless-microservice) was added to the pattern.

I also took the chance to fix the [location](https://github.com/cnoe-io/backstage-terraform-integrations/pull/27/commits/ae57e75725eca852b4e34985f7602a81e36c6680) for the Nvidia template which was wrong in the catalog.